BaristaSwift/Provider/Provider/View Controller/ViewController.swift

```diff
@@ -226,5 +226,10 @@ extension ViewController: UITableViewDataSource {
         let config = UISwipeActionsConfiguration(actions: [completeAction])
         return config
     }
+    
+    func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
```

Comment:
Unrelated cosmetic fix: the table view was showing the "delete" trailing swipe action (even though there is no handling code for it).